src/components/ShopList/index.js

```diff
@@ -31,7 +31,7 @@ class ShopList extends Component {
           longitude: position.coords.longitude,
         },
       });
-    }, console.error);
+    }, console.error());
```

Comment:
I think passing the `console.error` as a variable might be correct... this calls `console.error` with no arguments